sos/plugins/clear_containers.py

```diff
@@ -0,0 +1,80 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import re
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, \
+        SuSEPlugin
+
+
+class ClearContainers(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin,
+                      SuSEPlugin):
+    """Intel(R) Clear Containers configuration
+    """
+
+    plugin_name = 'clear_containers'
+    profiles = ('system', 'virt')
+
+    runtime = 'cc-runtime'
+    packages = (runtime,)
+
+    def attach_cc_config_files(self):
+        config_files = []
+
+        # start with the default file locations
+        config_files.append('/etc/clear-containers/configuration.toml')
+        config_files.append(
+                '/usr/share/defaults/clear-containers/configuration.toml'
+        )
```

Comment:
It's not a big deal but it might look neater to just place these in the static list initialisation, e.g.:

```python
        config_files = [
            '/etc/clear-containers/configuration.toml',
            '/usr/share/defaults/clear-containers/configuration.toml'
        ]
```
This is a pretty commonly used style for plugin metadata init.

Reply:
Fixed.